decidim-core/config/locales/en.yml

```diff
@@ -1279,6 +1279,29 @@ en:
         close_window: Close window
         share: Share
         share_link: Share link
+    statistics:
+      answers_count: Answers
+      assemblies_count: Assemblies
+      comments_count: Comments
+      debates_count: Debates
+      elections_count: Elections
+      endorsements_count: Endorsements
+      followers_count: Followers
+      headline: Statistics
+      meetings_count: Meetings
+      no_stats: There are no statistics yet.
+      orders_count: Supports
+      pages_count: Pages
+      participants_count: Participants
+      posts_count: Posts
+      processes_count: Processes
+      projects_count: Projects
+      proposals_count: Proposals
+      results_count: Results
+      supports_count: Supports
+      surveys_count: Surveys
+      users_count: Participants
+      votes_count: Supports
```

Comment:
This is linking the `decidim-core` module with all the others. I'd rather have these strings defined in each module's YAML files 🙏